### PR TITLE
Fix lm grid calculation

### PR DESCRIPTION
**Pull request by aknierim**
Fixes lm grid calculation in `pyvisgen.simulation.observation`. Subtracting source `ra` from the `ra` component of the `rd_grid` resulted in a wrong `l` component in the `lm_grid`.

### CODE REVIEW

**Comment by Kevin2** [comment]
is this ready for merge?

**Comment by aknierim** [comment]
I would say yes

File: pyvisgen/simulation/scan.py [hunk 1]
```diff
@@ -75,13 +75,13 @@ def calc_fourier(img, bas, lm, spw_low, spw_high):
         -2
         * pi
         * 1j
-        * (ul / c.value * spw_low + vm / c.value * spw_low + wn / c.value * spw_low)
+        * (ul + vm + wn) / 3e8 * spw_low
```
Comment (aknierim) [inline pyvisgen/simulation/scan.py:78]: We could change the value of c to the value given by `scipy.constants` though

Comment (Kevin2) [reply pyvisgen/simulation/scan.py]: done

**Comment by FeGeyer** [comment]
It still produces zeros and NaNs for me. I try to find out what the problem is

**Comment by FeGeyer** [comment]
The removal of the `ra` in `create_rd_grid` leads to values outside of the [-1, 1] range in the arcsin in `angularDistance`, which is the reason for the NaNs. Maybe we can speak on Monday and try to work out a solution that works for both Tom and me.
If I add the removed `ra`s again, it's working again.

**Comment by aknierim** [comment]
> Maybe we can speak on Monday and try to work out a solution that works for both Tom and me.

Sounds good to me. We should fix this as soon as possible

**Comment by aknierim** [comment]
> The removal of the `ra` in `create_rd_grid` leads to values outside of the [-1, 1] range in the arcsin in `angularDistance`, which is the reason for the NaNs. Maybe we can speak on Monday and try to work out a solution that works for both Tom and me. If I add the removed `ra`s again, it's working again.


Fixed in https://github.com/radionets-project/pyvisgen/pull/32/commits/eeaac6797b588d8bf7a2721064120c430fb433d3

**Comment by aknierim** [comment]
https://github.com/radionets-project/pyvisgen/pull/32/commits/add7e77869b6992c692764b0682304d9146a3411 works for me :+1:

**Review by FeGeyer — APPROVED** [review]

**Pull request merged**